README.md

```diff
@@ -18,14 +18,14 @@ Ensure your JSON file adheres to the following validation rules:
   - summary: A brief summary of the mod, must not exceed 120 characters.
```

Comment:
title: Title of the mod, must not exceed 10 characters.
->
title: Title of the mod, must not exceed 20 characters.